deploy/scripts/src/cri-orange/nino-check.ts

```diff
@@ -83,7 +82,18 @@ const csvData1: Nino[] = new SharedArray('csvDataNino', () => {
     })
 })
 
+export function setup(): void {
+  describeProfile(loadProfile)
+}
+
 export function ninoCheck(): void {
+  let imposterstarted = false
+  if (!imposterstarted) {
+    const response = imposter.handler()
+    console.log('Response status: ', response.status)
+    console.log('Response body: ', response.body)
+    imposterstarted = true
+  }
```

Comment:
```suggestion
    const response = imposter.handler()
    console.log('Response status: ', response.status)
    console.log('Response body: ', response.body)
```
This code is functionally the same as removing the `imposterstarted` variable and the if statement, as it will always run.

If you are trying to run this once at the beginning of the test I'd move this call to the `setup()` function, or if you want to run it once per VU the `imposterstarted` will need to be declared outside of the function scope.

Reply:
Yea I included this in the `setup()` function initially but I had a chat with Sandra and was advised that this needs to run periodically instead of only once. 
To avoid running this as many time as the actual test iterations, we may have a separate function for this which will be called like 10% or 20% of the total number of executed iterations

Reply:
Okay, well that approach will need to be implemented and the code smell fixed first before merging

Reply:
Updated the approach to make 10 concurrent request (as a batch request) in the `setUp` function and fixed the code smell.